arches_lingo/serializers.py

```diff
@@ -1,57 +0,0 @@
-from django.utils.translation import gettext as _
-from rest_framework.exceptions import ValidationError
-
-from arches.app.models.serializers import ArchesTileSerializer
-
-from arches_controlled_lists.datatypes.datatypes import ReferenceDataType
-from arches_controlled_lists.models import ListItem
-
-
-class LingoTileSerializer(ArchesTileSerializer):
-
-    def validate_appellative_status(self, data):
```

Comment:
An arches app developer will have a choice, either implement logic in their new APIs only, like we were doing before, or write an arches function that adds compatibility with the legacy resource editor.